scipy/signal/filter_design.py

```diff
@@ -4218,27 +4218,76 @@ def cheb2ap(N, rs):
 
 EPSILON = 2e-16
 
+# tolerance for the inverse function solvers used in ellipap
+_ELLIP_TOL = 1e-8
 
-def _vratio(u, ineps, mp):
-    [s, c, d, phi] = special.ellipj(u, mp)
-    ret = abs(ineps - s / c)
-    return ret
 
+def _f_vratio(u, ineps, mp):
+    """Function to solve in _solve_vratio"""
+    [s, c, *_] = special.ellipj(u, mp)
+    return c * ineps - s
```

Comment:
I don't have an opinion on whether this change is good or not

Reply:
This allows _solve_vratio to handle large ineps values (so, small eps values, and small rp values).  Using `ineps - s / c`:

```
>>> _solve_vratio(1e10, 0.5)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/scipy/scipy/signal/filter_design.py", line 4250, in _solve_vratio
    maxiter=1000)
  File "/home/scipy/scipy/optimize/_root_scalar.py", line 246, in root_scalar
    r, sol = methodc(f, a, b, args=args, **kwargs)
  File "/home/scipy/scipy/optimize/zeros.py", line 776, in brentq
    r = _zeros._brentq(f, a, b, xtol, rtol, maxiter, args, full_output, disp)
ValueError: f(a) and f(b) must have different signs
```

Having said that, for this particular example the supposedly better variant (`c * ineps - s`) gives the same answer for `ineps` values of `1e9` and greater.